Managers/LuaMan.cpp

```diff
@@ -403,6 +405,38 @@ namespace RTE {
 		return isDefined;
 	}
 
+	/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	const std::vector<std::string> & LuaMan::DirectoryList(const std::string &filePath) {
+		std::string fullPath = System::GetWorkingDirectory() + filePath;
+
+		m_Paths.clear();
+
+		for (const auto& entry : std::filesystem::directory_iterator{fullPath}) {
+			if (entry.is_directory()) {
+				m_Paths.push_back(entry.path().filename().string());
+			}
+		}
+
+		return m_Paths;
```

Comment:
Did you try just returning a newly made (in the method), non-reference vector of strings here instead of using the member variable?
I feel like we talked about this before and maybe there was no way to do it?

Reply:
Looked at the conversation in project coding, I guess you tried  a bunch of stuff with no luck.
I assume making a `new vector<...>` and returning that with `iterator` and an `adopt(_result)` policy didn't work out?

Reply:
> I assume making a new vector<...> and returning that with iterator and an adopt(_result) policy didn't work out?

![image](https://user-images.githubusercontent.com/32989873/204351824-fe395066-846e-48b9-9453-53f33410d801.png)
Doesn't work :(